server/services/Registry.js

```diff
@@ -13,17 +13,57 @@ type DbResponse<R> = {|
   rowsAffected: Array<number>,
 |};
 
+//should match SQL output
 export type DogLicense = {|
-  CertificateID: number,
-  'Registered Number': string,
-  InOut: 'I' | '*' | '#',
-  'Date of Death': ?string,
-  'Decedent Name': string,
-  'Last Name': string,
-  'First Name': string,
-  RegisteredYear: string,
-  AgeOrDateOfBirth: string,
-  Pending: number,
+  CertificateID: string,
+  'last_name': string,
+  'first_name': string,
+  'year': string,
+  'animal_id': string[],
```

Comment:
likewise, just drop a simple “join column” comment on these `[]` fields.

My philosophy on code comments is that they’re there not to describe what the code is doing (that’s what the code is for) but to explain things that do not match immediate expectations.